### PR TITLE
First version of Release Note for WAFS.v7.0.0

### PR DESCRIPTION
**Pull request by YaliMao-NOAA**
@aerorahul @HuiyaChuang-NOAA This is my first version of release note. Please feel free to edit it or let me know if you have any questions.
Here is the direct link of release note: https://github.com/NOAA-EMC/WAFS/blob/f5511627fe3f3915725eed103992ca608c7d6f4c/docs/Release_Notes.md

### CODE REVIEW

**Comment by aerorahul** [comment]
I'll review this after my meeting.  I have also asked @KateFriedman-NOAA to review as she is very good at gathering important information and documenting it in the release notes.

File: docs/Release_Notes.md [hunk 1]
```diff
@@ -31,60 +37,151 @@ Lastly, link the `ecflow` scripts by executing:
...
+ |   JWAFS_UPP         (new)   |                                                        |
 
 
 Parm Changes
 ------------
-
+1. Remove parm/wafs/legend
+2. parm/upp is created after system building. Per AWC request, WAFS UPP control files add 4 low levels for icing and 1 upper lower for turbulence:
+   - postxconfig-NT-GFS-WAFS.txt
+   -  postcntrl_gfs_wafs.xml
```
Comment (KateFriedman-NOAA) [inline docs/Release_Notes.md]:
```suggestion
   - postcntrl_gfs_wafs.xml
```

**Review by KateFriedman-NOAA — APPROVED** [review]
New release notes look good! Left one comment to remove an extra space.

File: docs/Release_Notes.md [hunk 2]
```diff
@@ -31,60 +37,151 @@ Lastly, link the `ecflow` scripts by executing:
...
+Filename changes according to EE2 standards:
+ |   wafs.v7                   |    GFSv16                     
+ |   ------------------------- |    ----------------------------------------------------|
+ |   JWAFS_GCIP                |<- JGFS_ATMOS_WAFS_GCIP                                 |
+ |   JWAFS_GRIB                |<- JGFS_ATMOS_WAFS                                      |
+ |   JWAFS_GRIB2_0P25          |<- JGFS_ATMOS_WAFS_GRIB2_0P25                           |
+ |   JWAFS_GRIB2_0P25_BLENDING |<- JGFS_ATMOS_WAFS_BLENDING_0P25                        |
+ |   JWAFS_GRIB2_1P25          |<- JGFS_ATMOS_WAFS_GRIB2                                |
+ |   JWAFS_GFS_MANAGER (new)   |   JGFS_ATMOS_WAFS_BLENDING (removed after cleaning up) |
+ |   JWAFS_UPP         (new)   |                                                        |
```
Comment (aerorahul) [inline docs/Release_Notes.md]:
```suggestion
 |         | JGFS_ATMOS_WAFS_BLENDING (removed after cleaning up) |
  |   JWAFS_GFS_MANAGER (new)   |   |
 |   JWAFS_UPP         (new)   |                                                        |
```

File: docs/Release_Notes.md [hunk 3]
```diff
@@ -31,60 +37,151 @@ Lastly, link the `ecflow` scripts by executing:
...
 --------------
+First of all, filename changes according to EE2 standards:
+ |   wafs.v7                       |    GFSv16
+ |   ----------------------------- |    ---------------------------------------------------------|
+ |   exwafs_gcip.sh                |<- exgfs_atmos_wafs_gcip.sh                                  |
+ |   exwafs_grib2_0p25_blending.sh |<- exgfs_atmos_wafs_blending_0p25.sh                         |
+ |   exwafs_grib2_0p25.sh          |<- exgfs_atmos_wafs_grib2_0p25.sh                            |
+ |   exwafs_grib2_1p25.sh          |<- exgfs_atmos_wafs_grib2.sh                                 |
+ |   exwafs_grib.sh                |<- exgfs_atmos_wafs_grib.sh                                  |
+ |   exwafs_gfs_manager.sh (new)   |   exgfs_atmos_wafs_blending.sh (removed after cleaning up)  |
```
Comment (aerorahul) [inline docs/Release_Notes.md]:
```suggestion
|     |  exgfs_atmos_wafs_blending.sh (removed after cleaning up) |
 |   exwafs_gfs_manager.sh (new)   |     |
```

File: docs/Release_Notes.md [hunk 4]
```diff
@@ -31,60 +37,151 @@ Lastly, link the `ecflow` scripts by executing:
...
 Fix Changes
 -----------
-
+1. Remove fix/wafs/legend folder
+2. Remove files under fix/wafs:
+   - grib2_blended_wafs_wifs_fFF.0p25
+   - grib2_gfs_wafs_wifs_fFF.0p25
+   - grib_wafsgfs_intdsk
+   - grib_wafsgfs_intdskf00   
+3. Under fix/wafs, filenames are changed according to EE2 standards.
```
Comment (aerorahul) [inline docs/Release_Notes.md]: fix files do not have a EE2 naming convention

File: docs/Release_Notes.md [hunk 5]
```diff
@@ -31,60 +37,151 @@ Lastly, link the `ecflow` scripts by executing:
...
+  * Inside WAFS, there are subfolders categoried by job names
+    * |-- upp
+    * |-- gcip
+    * |-- grib
+    * |-- grib2
+    * |----- 1p25
+    * |----- 0p25
+    * |-------- blending
+* Files to be retired
+  * `gfs.tCCz.wafs_icao.grb2fFFF`
```
Comment (aerorahul) [inline docs/Release_Notes.md:148]: @KateFriedman-NOAA 
Does this need to be captured in the GFS release notes from the wafs separation?

File: docs/Release_Notes.md [hunk 6]
```diff
@@ -31,60 +37,151 @@ Lastly, link the `ecflow` scripts by executing:
...
+* Filename changes
+  * Renamed according to EE2 implementation standards
+  * Exceptions: files sent to UK
+  * Refer to this document: https://docs.google.com/spreadsheets/d/1Nt343Z9x9UycweFik3HRFpXkqIjs7m20s15yGOhsgUY
```
Comment (aerorahul) [inline docs/Release_Notes.md]: IMO, the name changes should be summarized in the release notes.  The google document can change, and will need access updated for anyone who looks at this release note.

File: docs/Release_Notes.md [hunk 7]
```diff
@@ -31,60 +37,151 @@ Lastly, link the `ecflow` scripts by executing:
...
 Dissemination Information
 -------------------------
+* Filename changes and dbn_alert subtype changes
+  * Refer to this document: https://docs.google.com/spreadsheets/d/1Nt343Z9x9UycweFik3HRFpXkqIjs7m20s15yGOhsgUY
+* Where should this output be sent?
+  * Same as current operations in GFS WAFS
+* Who are the users?
+  * Same as current operations WAFS users
+* Which output files should be transferred from PROD WCOSS to DEV WCOSS?
+  * Same as current operations in GFS WAFS
```
Comment (aerorahul) [inline docs/Release_Notes.md]: If the list is not too long, it would be good to document that here.

**Review by aerorahul — CHANGES_REQUESTED** [review]
some feedback

File: docs/Release_Notes.md (hunk 5 above)
Comment (KateFriedman-NOAA) [reply docs/Release_Notes.md:148]: Probably. We'll need to list all WAFS files that will be removed from GFS COM in the release notes. I can note that this one file is being retired as well.

Comment (aerorahul) [reply docs/Release_Notes.md:148]: @YaliMao-NOAA 
Can you provide a list of files that the GFS will not be producing and their equivalent WAFS files so @KateFriedman-NOAA can include it in the GFS release notes?

Comment (YaliMao-NOAA) [reply docs/Release_Notes.md:148]: @KateFriedman-NOAA Here is the list of WAFS files which are going to be moved out of GFS.
https://docs.google.com/spreadsheets/d/1Nt343Z9x9UycweFik3HRFpXkqIjs7m20s15yGOhsgUY

File: docs/Release_Notes.md (hunk 7 above)
Comment (YaliMao-NOAA) [reply docs/Release_Notes.md]: @aerorahul Are you talking about line 162, or line 162, 160 and 158? I can answer 162.
As to questions on line 158 and 160, @HuiyaChuang-NOAA, should NCO dataflow be the right contact to answer these questions?

Comment (aerorahul) [reply docs/Release_Notes.md]: I am referring to the whole section between lines 160-165

File: docs/Release_Notes.md (hunk 4 above)
Comment (YaliMao-NOAA) [reply docs/Release_Notes.md]: I will remove the words of "EE2 standards", while keeping fix/wafs filename changes

File: docs/Release_Notes.md [hunk 8]
```diff
@@ -31,60 +37,153 @@ Lastly, link the `ecflow` scripts by executing:
...
+    * wafs.tCCz.master.fFFF.grib2 when FFF<=048
+    * grib2/0p25/wafs.tCCz.awf.0p25.fFFF.grib2 when FFF<=036
 
 
 Dissemination Information
 -------------------------
+* Filename changes and dbn_alert subtype changes
+  * Refer to this document: https://docs.google.com/spreadsheets/d/1Nt343Z9x9UycweFik3HRFpXkqIjs7m20s15yGOhsgUY
+* Where should this output be sent?
+  * Same as current operations in GFS WAFS
```
Comment (HuiyaChuang-NOAA) [inline docs/Release_Notes.md:187]: Same as current operations in GFS WAFS and refer to https://docs.google.com/spreadsheets/d/1Aq1d1uCr5RX9JvoR3B_pi21dY2ndvvHvmAdjYHAgAQg/edit?gid=0#gid=0

File: docs/Release_Notes.md (hunk 7 above)
Comment (HuiyaChuang-NOAA) [reply docs/Release_Notes.md]: > I am referring to the whole section between lines 160-165

Shouldn't Dataflow be the one answering 160?  Nevertheless,  I added a link from Dataflow

File: docs/Release_Notes.md [hunk 9]
```diff
@@ -31,60 +37,153 @@ Lastly, link the `ecflow` scripts by executing:
...
+* Who are the users?
+  * Same as current operations WAFS users
```
Comment (HuiyaChuang-NOAA) [inline docs/Release_Notes.md]: AWC, UK Met Office, SPC, and ICAO subscribed users

File: docs/Release_Notes.md [hunk 10]
```diff
@@ -31,60 +37,153 @@ Lastly, link the `ecflow` scripts by executing:
...
+* Which output files should be transferred from PROD WCOSS to DEV WCOSS?
+  * Same as current operations in GFS WAFS
```
Comment (HuiyaChuang-NOAA) [inline docs/Release_Notes.md]: All WAFS files should be transferred

**Pull request merged**